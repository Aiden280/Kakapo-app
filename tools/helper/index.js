import fs from 'fs-extra';

// Upload stats.json to `http://webpack.github.io/analyse` for analysis.
// Also you can analyze size with `https://github.com/robertknight/webpack-bundle-size-analyzer`:
// `cat stats.json | analyze-bundle-size`
export const writeStats = stats =>
  fs.writeFile('stats.json', JSON.stringify(stats.toJson(), null, 4));

export default {};
