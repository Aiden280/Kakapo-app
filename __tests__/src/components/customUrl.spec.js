import React from 'react';
import { shallow } from 'enzyme';
import { getData } from '../helper';
import { ImportCustomUrl as CustomUrl } from 'components/';

function setup(props = {}) {
  const propData = {
    ...getData('intl'),
    ...getData('themes'),
    ...props
  };
  return { props, wrapper: shallow(<CustomUrl {...propData} />) };
}

test('<CustomUrl/>', () => {
  expect.assertions(2);
  const { wrapper } = setup();
  expect(wrapper.type()).toBe('div');
  expect(wrapper.prop('className')).toBe('customurl');
});
