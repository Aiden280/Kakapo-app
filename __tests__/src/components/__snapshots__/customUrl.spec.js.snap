// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomUrl/> 1`] = `
<div
  className="customurl"
>
  <h5>
    Add a custom stream
  </h5>
  <form
    onSubmit={[Function]}
  >
    <div
      className="media-import"
    >
      <div
        className="group"
      >
        <input
          autoComplete="off"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
        />
        <label
          htmlFor="name"
        >
          Choose a name for this sound
        </label>
        <span
          className="highlight"
        />
        <span
          className="bar"
        />
      </div>
      <div
        className="group"
      >
        <input
          autoComplete="off"
          name="url"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
        />
        <label
          htmlFor="url"
        >
          Enter a link to an audio file
        </label>
        <span
          className="highlight"
        />
        <span
          className="bar"
        />
      </div>
      <button
        className="button"
        style={
          Object {
            "backgroundColor": "#4CAF50",
            "borderColor": "#4CAF50",
          }
        }
        type="button"
      >
        Save
      </button>
    </div>
  </form>
</div>
`;
